### ISSUE TITLE
[High Contrast] Slider is coming as a circle only in high contrast modes in IE/Edge

### ISSUE THREAD

**Issue by sulabh-msft** [OfficeDev/office-ui-fabric-react#1640]
### Bug Report

- __Package version(s)__: (2.20.2)
- __Browser and OS versions__: (IE/Edge Win10)
Microsoft Edge 40.15063.0.0
Microsoft EdgeHTML 15.15063

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No)
No

Requested priority: (Blocking, High, Normal, Low) 
High

Products/sites affected: (if applicable)
VSTS or any site using slider  ( and users using it high contrast modes )

#### Describe the issue:
Slider: https://dev.office.com/fabric#/components/slider

Slider in non-high contrast mode
![image](https://cloud.githubusercontent.com/assets/13728069/25488221/1c7a8406-2b84-11e7-9100-94e1b650557d.png)

In "High contrast Black" and "High contrast White" [ could be issue in other high contrast modes as well] Slider is not visually similar to what we see in non-high contrast mode, Only the silder knob(circle) is visible in this mode .

#### Actual behavior:
High Contrast Black
Slider: https://dev.office.com/fabric#/components/slider
![image](https://cloud.githubusercontent.com/assets/13728069/25486847/0a9009a4-2b80-11e7-90b4-516a239d9bfd.png)

High Contrast White
![image](https://cloud.githubusercontent.com/assets/13728069/25486893/2ac6bf9c-2b80-11e7-9181-6f8fcfd992e3.png)

#### Expected behavior:
In high contrast mode we should get the slider visually similar to what we get in non-high contrast mode.

#### User Impact:
User with low vision who relay on screen reader will face difficulty to access the slider.
